src/fusor/examples/bcr_abl1.json

```diff
@@ -1,111 +1,131 @@
+    "adjoinedSequences": [{
+      "type": "SequenceLocation",
+      "sequenceReference": {
+        "id": "GRCh38:chr22",
+        "type": "SequenceReference",
+        "refgetAccession": "SQ.7B7SHsmchAR0dFcDCuSFjJAo7tX87krQ",
+        "residueAlphabet": "na"
       },
-      "element_genomic_end": {
-        "id": "fusor.location_descriptor:NC_000022.11",
-        "type": "LocationDescriptor",
-        "label": "NC_000022.11",
-        "location": {
-          "type": "SequenceLocation",
-          "sequence_id": "refseq:NC_000022.11",
-          "interval": {
-            "type": "SequenceInterval",
-            "start": {
-              "type": "Number",
-              "value": 23253980
-            },
-            "end": {
-              "type": "Number",
-              "value": 23253981
+      "end": 23290413,
+      "extensions": [
+        {
+          "name": "NM_004327.4:e._14",
+          "description": "VICC exon representation of the aligned transcript boundary.",
+          "value": {
+            "exon_end": 14,
+            "exon_end_offset": 0,
+            "sequenceReference":{
+              "type": "SequenceReference",
+              "id": "NM_004327.4",
+              "refgetAccession": "SQ.kpytJsXw3BwLC3oBSjHQS1kwxs4WO3I3",
+              "residueAlphabet": "na"
             }
           }
-        }
-      },
-      "exon_end": 2,
-      "exon_end_offset": 182
-    },
-    {
-      "type": "LinkerSequenceElement",
-      "linker_sequence": {
-        "id": "sequence:ACTAAAGCG",
-        "type": "SequenceDescriptor",
-        "sequence": "ACTAAAGCG",
-        "residue_type": "SO:0000348"
-      }
-    },
-    {
-      "type": "TranscriptSegmentElement",
-      "transcript": "refseq:NM_005157.5",
-      "exon_start": 2,
-      "exon_start_offset": -173,
-      "gene_descriptor": {
-        "id": "normalize.gene:ABL1",
-        "type": "GeneDescriptor",
-        "label": "ABL1",
-        "gene_id": "hgnc:76"
-      },
-      "element_genomic_start": {
-        "id": "fusor.location_descriptor:NC_000009.12",
-        "type": "LocationDescriptor",
-        "label": "NC_000009.12",
-        "location": {
-          "type": "SequenceLocation",
-          "sequence_id": "refseq:NC_000009.12",
-          "interval": {
-            "type": "SequenceInterval",
-            "start": {
-              "type": "Number",
-              "value": 130854064
+        },
+        {
+          "name": "NM_004327.4:c._2782",
+          "description": "Transcript SequenceLocation of the aligned transcript boundary.",
+          "value": {
+            "type": "SequenceLocation",
+            "sequenceReference": {
+              "id": "NM_004327.4",
+              "type": "SequenceReference",
+              "refgetAccession": "SQ.kpytJsXw3BwLC3oBSjHQS1kwxs4WO3I3",
+              "residueAlphabet": "na"
             },
-            "end": {
-              "type": "Number",
-              "value": 130854065
-            }
+            "end": 3234
```

Comment:
Here, we again use `end`, as we always expect to see in the transcript sequence representation for the 5' partner. For transcripts aligning to the negative strand of a chromsome, this would remain `end`, even though the chromosome representation (on line 13) would use `start`.